tftrt/blog_posts/Leveraging TensorFlow-TensorRT integration for Low latency Inference/tf2_inference.py

```diff
@@ -0,0 +1,148 @@
+import os
+import copy
+import argparse
+import time
+
+from statistics import mean
+
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = "3"
+
+import tensorflow as tf
+from tensorflow.python.compiler.tensorrt import trt_convert as trt
+
+SAVEDMODEL_PATH = "./checkpoints/saved_model"
+
+def load_with_converter(path, precision, batch_size):
+    """ Load a saved model using a TF-TRT converter, and return the converter """
```

Comment:
If the coding style is similar to tensorflow, then we need to:
s/Load/Loads/
s/return/returns/

Reply:
Here I just copy/pasted the exact code used in the blog post.
If we want to change anything, we need to change it on both side.

Reply:
Are you sure about this? I couldn't find the code in the blog post, it only has a placeholder for a link to an external script. This PR is trying to provide such a link.

Reply:
Oh right, I push the fix & merge since it was your only remark